code/go/0chain.net/blobbercore/handler/protocol.go

```diff
@@ -119,80 +133,47 @@ func BlobberHealthCheck(ctx context.Context) (string, error) {
 	if config.Configuration.Capacity == 0 {
 		return "", ErrBlobberHasRemoved
 	}
-	txn, err := transaction.NewTransactionEntity()
-	if err != nil {
-		return "", err
-	}
-	Logger.Info("Blobber health check to the blockchain.")
-	err = txn.ExecuteSmartContract(transaction.STORAGE_CONTRACT_ADDRESS,
-		transaction.BLOBBER_HEALTH_CHECK, "", 0)
-	if err != nil {
-		Logger.Info("Failed during blobber health check to the mining network",
-			zap.String("err:", err.Error()))
-		return "", err
-	}
 
-	return txn.Hash, nil
-}
-
-func UpdateBlobberSettings(ctx context.Context) (string, error) {
 	txn, err := transaction.NewTransactionEntity()
 	if err != nil {
 		return "", err
 	}
 
-	sn, err := getStorageNode()
-	if err != nil {
-		return "", err
-	}
-
-	snBytes, err := json.Marshal(sn)
-	if err != nil {
-		return "", err
-	}
+	Logger.Info("Health check on the blockchain")
```

Comment:
This log does not look particularly useful.

Reply:
ok, lets drop it.
i'm 50/50 about it.